### ISSUE TITLE
.shortnameToUnicode() injects weird symbol at the end of the string

### ISSUE THREAD

**Issue by suprMax** [Ranks/emojione#290]
Quick visual steps to reproduce:
1. Go to http://git.emojione.com/demos/latest/shortnametounicode.html
2. Enter `Hello world! :smile: :tools:`, convert, copy output
3. Go to http://git.emojione.com/demos/latest/jstoimage.html
4. Paste copied text, convert. You will get `🛠️` that is not converted.

This is what you see if you decode those chars:

```
> p.ucs2.decode('🛠') // normal emoji, just copied
[ 128736 ]
> p.ucs2.decode('🛠️') // passed through encode-decode cycle
[ 128736, 65039 ]
```

I'm no unicode expert, but something shady is going on here.

P.S. I realize that you might think it's a copy-paste issue. It's not, using direct IO of those methods:

```
e.toImage(e.shortnameToUnicode('Hello world! :smile: :tools:'))
'Hello world! <img class="emojione" alt="😄" src="//cdn.jsdelivr.net/emojione/assets/png/1f604.png?v=2.2.5"/> 🛠️'
```

```
> p.ucs2.decode(e.shortnameToUnicode(':smile::tools:'))
[ 128516, 128736, 65039 ]
```

(all logs `p` is standart node punycode and `e` is emojione v2.2.5)

**Comment by caseyahenson** [comment]
We've been a little behind but will be looking into this in the next few days, sorry for the delay!

**Comment by caseyahenson** [comment]
@suprMax It looks like what's happening here is that you're getting a trailing character from the "unicode alt" value for :tools: (1f6e0-fe0f). I'll do some reworking on this and get it straightened out.

**Comment by suprMax** [comment]
Yup I figured it's related to one (some?) emoji, b/c I could only reproduce it with `:tools:`. Worth checking others though, I suppose. Thanks for fixing this! 👍

**Comment by caseyahenson** [comment]
This should be fixed in the 2.2.7 release, if you're still running into issues let me know!
